docs/GettingStarted.md

```diff
@@ -448,71 +382,42 @@ For example, to change the default service name:
       config.server_middleware do |chain|
         chain.add(
           Datadog::Contrib::Sidekiq::Tracer,
-          sidekiq_service: 'sidekiq-notifications'
+          sidekiq_service: 'sidekiq-notifications',
+          tracer: Datadog.tracer
```

Comment:
Sidekiq doesn't have our fancy `Datadog.configure`?

Reply:
It doesn't. I'll open a PR for that.

Reply:
PR available at: https://github.com/DataDog/dd-trace-rb/pull/294

Reply:
@p-lambert merged! you can update this PR then! thank you!